superset-frontend/packages/superset-ui-core/src/components/index.ts

```diff
@@ -17,5 +17,160 @@
  * under the License.
  */
 
+/*
+ * Re-exporting of components in @superset-ui/core/components to facilitate
+ * their imports by other components.
+ * E.g. import { Select } from '@superset-ui/core/components', probably in the future from '@superset-ui/components'
+ */
+export * from './Input';
+export { AntdThemeProvider } from './AntdThemeProvider';
+export {
+  ConfirmStatusChange,
+  type ConfirmStatusChangeProps,
+} from './ConfirmStatusChange';
+export { Alert, type AlertProps } from './Alert';
+export { CertifiedBadge } from './CertifiedBadge';
+export * from './Icons';
+export * from './Timer';
+export {
```

Comment:
Is it now a good time to go with the approach that we wanted to implement where we are just re-exporting everything from here?

`export * from '{component}';`